x-pack/legacy/plugins/alerting_ui/np_ready/public/application/sections/actions_list/components/create_menu_popover.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { useContext, useState } from 'react';
+import {
+  EuiSpacer,
+  EuiText,
+  EuiFlexItem,
+  EuiIcon,
+  EuiFlexGroup,
+  EuiButton,
+  EuiPopover,
+  EuiContextMenuPanel,
+  EuiContextMenuItem,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { ActionsContext } from '../../../context/actions_context';
+
+interface Props {
+  actionTypes: any;
+}
+
+export const AlertingActionsDropdown: React.FunctionComponent<Props> = ({ actionTypes }) => {
+  const { addAction } = useContext(ActionsContext);
+
+  const [isPopoverOpen, setIsPopOverOpen] = useState<boolean>(false);
+
+  const actionTypesSettings = (val: string) => {
+    let res;
+    switch (val) {
+      case '.email':
+        res = {
+          iconClass: 'email',
+          selectMessage: i18n.translate(
+            'xpack.alertingUI.sections.actions.emailAction.selectMessageText',
+            {
+              defaultMessage: 'Send an email from your server.',
```

Comment:
This file is for the next upcoming PR (https://github.com/elastic/kibana/pull/48191) to create / edit actions. We decided instead of cleaning it up that we'll leave it there until the next PR lands. In order to avoid merge conflicts, we'll address this feedback in the next PR cc @YulNaumenko.